main/conceptual/getting-started/start-a-project.md

```diff
@@ -1,16 +1,12 @@
 # Starting a Project
 
-::: tip Beta status
-The Agoric platform is at the beta stage. It is in the process of being formally evaluated for vulnerabilities, and is undergoing security treatment and hardening to support mainnet-1 launch. Do not use for production purposes. 
-:::
-
-Before we begin, you should use `agoric --version` to double-check that you have [installed the Agoric SDK](/getting-started/before-using-agoric.md).
+Before we begin, you should use `agoric --version` to double-check that you have [installed the Agoric SDK](./README.md).
 
 If it is available, then you successfully installed the Agoric SDK. If not, then please do so before continuing. 
 
-After you've [installed the Agoric SDK](/getting-started/before-using-agoric.md), then you're ready for your first _Agoric Dapp_ (decentralized application) by continuing the instructions below. 
+After you've [installed the Agoric SDK](./README.md), then you're ready for your first _Agoric Dapp_ (decentralized application) by continuing the instructions below. 
 
-We'll be running **three terminal windows**. See below: 
+We'll be running three terminal windows. See below: 
```

Comment:
That emphasis seemed reasonably important, based on feedback we've gotten from people using the docs.

Reply:
K. I put the bold formatting back in. 

(I don't like how this page reads, but it's not worth spending the time right now to revise it. I'll come back to this topic sometime in the future.)